assets/scripts/docs.js

```diff
@@ -40,16 +40,21 @@ $(function() {
     generateApiClassNavItems();
   }
 
-  var clipboard = new Clipboard('[data-clipboard-text]');
-  clipboard.on('success', function(e) {
-    var previousText = $(e.trigger).text();
-    $(e.trigger).text('Copied!');
-    setTimeout(function() {
-      $(e.trigger).text(previousText);
-    }, 2000);
-  });
-  clipboard.on('error', function(e) {
-    $(e.trigger).text('Press ⌘-C now to copy');
-  });
+  if(document.queryCommandSupported('copy')) {
+    var clipboard = new Clipboard('[data-clipboard-text]');
+    clipboard.on('success', function(e) {
+      var previousText = $(e.trigger).text();
+      $(e.trigger).text('Copied!');
+      setTimeout(function() {
+        $(e.trigger).text(previousText);
+      }, 2000);
+    });
+    clipboard.on('error', function(e) {
+      $(e.trigger).text('Press ⌘-C now to copy');
```

Comment:
Does using the symbol have any support issues? We can use `&#8984;` [Reference](http://www.interactiveoctopus.com/blog/index.php/2008/01/display-the-apple-command-symbol-in-html/)